### PR TITLE
use descriptive names for maps, ROIs, and lists

### LINKED ISSUE #55: XPCS subentry for maps and lists - fix plain text representations
- [x] dqlist --> dynamic_q_list
- [x]  sqlist --> static_q_list
- [x]  dphillist --> dynamic_phi_list,    This one has a mask/map, but is associated with `dynamic_roi_map`
- [x]  sphillist --> static_phi_list
- [x]  dqmap --> dynamic_roi_map
- [x]  sqmap --> static_roi_map

maybe some description of 
"roi" are for computers/code, "q" and `phi` is for humans, scientists, users.  phi is only used for SAXS like experiments with  where constant radii are chopped into pie slices (azimuthal binning).

### PR DESCRIPTION
**Pull request by prjemian**
FIX #55

### CODE REVIEW

File: tests/nexus_aps_map.json [hunk 1]
```diff
@@ -20,10 +20,10 @@
...
-	"/entry/XPCS/instrument/mask/sqmap": "/xpcs/sqmap",
-	"/entry/XPCS/instrument/mask/sqlist": "/xpcs/sqlist",
-	"/entry/XPCS/instrument/mask/sphilist": "/xpcs/sphilist"
+	"/entry/XPCS/instrument/mask/dynamic_roi_map": "/xpcs/dqmap",
+	"/entry/XPCS/instrument/mask/dynamic_q_list": "/xpcs/dqlist",
+	"/entry/XPCS/instrument/mask/dynamic_phi_list": "/xpcs/dynamic_phi_list",
+	"/entry/XPCS/instrument/mask/static_roi_map": "/xpcs/sqmap",
+	"/entry/XPCS/instrument/mask/static_q_list": "/xpcs/static_q_list",
+	"/entry/XPCS/instrument/mask/static_phi_list": "/xpcs/sphilist"
 }
```
Comment (JulReinhardt) [inline tests/nexus_aps_map.json]: I think this json file might be obsolete anyways, but to be consistent, the right side of the mapping would still need to be dphilist and sqlist as these come from the original APS files, where the path is named like this.

**Review by JulReinhardt — APPROVED** [review]
I've made the minor changes as in my comment on the json file. Ready to merge IMHO

**Review by AbbyGi — APPROVED** [review]

**Comment by prjemian** [comment]
@ambarb @AZjk Ready for your review

**Review by AZjk — APPROVED** [review]

**Review by ambarb — APPROVED** [review]
this PR is about names and not doc strings so I am not commenting on docstrings

**Pull request merged**